### PR TITLE
Bump postcss-load-config

### PR DESCRIPTION
**Pull request by benmccann**
Closes https://github.com/sveltejs/svelte-preprocess/issues/306

(the lint failure is unrelated to this PR)

### CODE REVIEW

**Comment by dummdidumm** [comment]
Could this be `2 | 3` (not sure about the exact syntax)? After all, it's not required for people to use version 3, only if they want cjs support.

**Review by tanhauhau — APPROVED** [review]

**Comment by benmccann** [comment]
Is there a compelling reason to want people to be able to use v2? Normally I see us supporting multiple versions for things like Typescript or Postcss where there may be some difficulty in upgrading, but this one seems so easy to upgrade

**Comment by dummdidumm** [comment]
`peerDependencies` means "user, please install this on your own", and `svelte-preprocess` will work perfectly fine with `postcss-load-config` 2.x . Only if you want to use SvelteKit, you need version 3.

**Comment by benmccann** [comment]
Yeah, but I don't know why anyone should want to stay on version 2

**Comment by dummdidumm** [comment]
Maybe someone uses a PostCSS plugin that needs 2.x behavior (don't know what changed for 3.x). Making `peerDependencies` accept more versions better describes the "I don't really care which version you use"-notion of `svelte-preprocess`. It's SvelteKit which needs 3.x to work, not `svelte-preprocess`.

**Comment by benmccann** [comment]
It makes sense to me to support multiple postcss versions. This isn't actually the postcss version though, which is specified as `^7 || ^8`. It's just a library that reads your `postcss.config.js` file from disk if I understand correctly. The only breaking change is that it requires Node 10, which we already require anyway: https://github.com/postcss/postcss-load-config/blob/master/CHANGELOG.md#breaking-changes

**Review by dummdidumm — APPROVED** [review]
If that's the only breaking change then I'm okay with that.

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "coffeescript": "^2.5.1",
     "less": "^3.11.3",
     "postcss": "^7 || ^8",
-    "postcss-load-config": "^2.1.0",
+    "postcss-load-config": "^3.0.0",
```
Comment (kaisermann) [inline package.json:103]:
```suggestion
    "postcss-load-config": "^2.1.0 || ^3.0.0",
```

**Review by kaisermann — APPROVED** [review]

**Comment by kaisermann** [comment]
Thanks! I've added v2 to the peerDep as well (as @dummdidumm rightly suggested). Released in 4.6.6 :tada:

**Pull request merged**